lib/importer.rb

```diff
@@ -38,21 +38,39 @@ def lookup_country(country_str)
   end
 
   def normalize_country(country_str)
```

Comment:
Any suggestion on how to make this long (and possibly ever growing) method better?